models/reporter.go

```diff
@@ -365,11 +367,23 @@ func (r *Reporter) SelectFilters(ctx context.Context, periodStartFrom, periodSta
 	if err != nil {
 		return nil, fmt.Errorf("cannot select client hosts dimension:%v", err)
 	}
-	err = r.db.SelectContext(ctx, &labels, queryLabels, periodStartFrom, periodStartTo)
+
+	rows, err := r.db.Query(queryLabels, periodStartFrom, periodStartTo)
```

Comment:
So that was the problem? sqlx just can't select labels properly?

Reply:
I don't know why. But yes, somehow it has conflict with insert data.